code/modules/mob/living/carbon/human/human.dm

```diff
@@ -141,23 +141,7 @@
 			if (prob(50))
 				Paralyse(10)
 
-	var/update = 0
-	for(var/obj/item/organ/limb/temp in organs)
-		switch(temp.name)
-			if("head")
-				update |= temp.take_damage(b_loss * 0.2, f_loss * 0.2)
-			if("chest")
-				update |= temp.take_damage(b_loss * 0.4, f_loss * 0.4)
-			if("l_arm")
-				update |= temp.take_damage(b_loss * 0.05, f_loss * 0.05)
-			if("r_arm")
-				update |= temp.take_damage(b_loss * 0.05, f_loss * 0.05)
-			if("l_leg")
-				update |= temp.take_damage(b_loss * 0.05, f_loss * 0.05)
-			if("r_leg")
-				update |= temp.take_damage(b_loss * 0.05, f_loss * 0.05)
-	if(update)
-		update_damage_overlays(0)
+	take_overall_damage(b_loss,f_loss)

```

Comment:
This is actually an error in calcualtions; basically when hands+feet were removed, they weren't pushed into right/left leg and right/left arm and just out and out dropped--it makes no sense to have a 100% damage value, and have 20% of it go in no mans lands when you can just reduce up front damage and have 100% of it distributed out properly.

You can look as far back as 4407 to see this born out.